src/containers/Header/Search.jsx

```diff
@@ -17,16 +17,30 @@ import {
   HASH_REGEX,
 } from '../shared/utils'
 import './search.scss'
+import { getNFTInfo } from '../../rippled/lib/rippled'
+import SocketContext from '../shared/SocketContext'
 
-const getIdType = (id) => {
+const isNFTOrTransactions = async (id, rippledContext) => {
+  try {
+    await getNFTInfo(rippledContext, id)
```

Comment:
Is it faster to look up an nft or a transaction?  We should choose the faster one.  At somepoint there should be a check to see if the hash matches both a transaction and an nft.

Reply:
will investigate 👍